packages/cli/test/integration/credentials/credentials.api.test.ts

```diff
@@ -370,6 +372,9 @@ describe('GET /credentials', () => {
 
 		expect(teamCredAsViewer.id).toBe(teamCredentialAsViewer.id);
 		expect(teamCredAsViewer.data).toBeDefined();
+		expect(
+			(teamCredAsViewer.data as unknown as ICredentialDataDecryptedObject).oauthTokenData,
+		).toBe(true);
```

Comment:
For my own clarity, this is testing that we no longer return `oauthTokenData` but we're just returning `true` for it instead as the FE needs it?

Reply:
Yes